src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -101,19 +101,19 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   /**
    * Parses message text and changes BBCode tags on styled text to HTML tags
    */
-  public String convertBBCode(String text){
-      String temp = text;
-
-      Map<String , String> bbMap = new HashMap<String , String>();
-      bbMap.put("\\[b\\](.+?)\\[/b\\]", "<strong>$1</strong>");
-      bbMap.put("\\[i\\](.+?)\\[/i\\]", "<em>$1</em>");
-      bbMap.put("\\[u\\](.+?)\\[/u\\]", "<u>$1</u>");
-      bbMap.put("\\[s\\](.+?)\\[/s\\]", "<del>$1</del>");
-
-      for (Map.Entry entry: bbMap.entrySet()) {
-        temp = temp.replaceAll(entry.getKey().toString(), entry.getValue().toString());
-      }
-      return temp;
-
-  }
+ // public String convertBBCode(String text){
+  //    String temp = text;
```

Comment:
Either delete the comments or make block comment and explain why the legacy code should be kept for future reference